oauth2/fosite_store_helpers_test.go

```diff
@@ -94,29 +48,33 @@ var defaultIgnoreKeys = []string{
 	"client.client_secret",
 }
 
-var defaultRequest = fosite.Request{
```

Comment:
We reference this variable in quite a lot of DB queries, it's a wonder there aren't more inter-test leaks.